modules/node-management.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture.adoc
+
+[id='node-management-{context}']
+= Node management in fully-managed {product-title}
+
+The fully-managed version of {product-title} version 4.0 integrates management of
+the container operating system and cluster management. Because the cluster manages
+both its updates and updates to RHCOS, {product-title} provides an opinionated
```

Comment:
> will you give me a little more context about why the RHCOS updates should be considered a subset of the OCP updates?

There's some more background in openshift/machine-config-operator#183 and openshift/installer#987.  Things are still a bit up in the air, but, as I understand it, RHCOS updates will eventually be triggered by update-payload bumps gated by a Cincinnati graph like all other cluster updates.

Reply:
OK. For now, I'm going to change it to "Because the cluster manages
its updates, including updates to Red Hat CoreOS on cluster nodes," because you're right, they're treated the same. I think it's helpful to emphasize that the RHCOS updates are managed here, too. It's helpful context.